### PR TITLE
New package: Samsung.UserManual.Series3 version 1.4.0.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111996&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/UserManual/Series3/1.4.0.0/Samsung.UserManual.Series3.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=5009457&CDCttType=SW&ModelType=C&ModelName=DP300A2A-A01US&VPath=SW/201209/20120926134248625/User_Manual_1.4.0.0.ZIP
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Inst.exe
+  InstallerSha256: 04C378551FAD44F3092A46948C352AD16AA6615601B22209E2E6F63BA21F6B31
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /passive
```
Comment (Trenly) [inline manifests/s/Samsung/UserManual/Series3/1.4.0.0/Samsung.UserManual.Series3.installer.yaml:16]:
```suggestion
  NestedInstallerType: burn
  NestedInstallerFiles:
  - RelativeFilePath: Inst.exe
  InstallerSha256: 04C378551FAD44F3092A46948C352AD16AA6615601B22209E2E6F63BA21F6B31
```

File: manifests/s/Samsung/UserManual/Series3/1.4.0.0/Samsung.UserManual.Series3.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.UserManual.Series3
+PackageVersion: 1.4.0.0
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=5009457&CDCttType=SW&ModelType=C&ModelName=DP300A2A-A01US&VPath=SW/201209/20120926134248625/User_Manual_1.4.0.0.ZIP
```
Comment (Trenly) [inline manifests/s/Samsung/UserManual/Series3/1.4.0.0/Samsung.UserManual.Series3.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201209/20120926134248625/User_Manual_1.4.0.0.ZIP
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Install doesn't actually install anything
image

**Comment by stephengillie** [comment]
This paackage seemed to partially install the application, leaving several setup files:
image
image
image

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**